src/cljs/main/broadfcui/common/entity_table.cljs

```diff
@@ -67,9 +67,11 @@
           (nil? entity-metadata) [:div {:style {:textAlign "center"}} [comps/Spinner {:text "Retrieving entity types..."}]]
           :else
           (let [attributes (map keyword (get-in entity-metadata [(keyword selected-entity-type) :attributeNames]))
-                attr-col-width (->> attributes count (/ 1000) int (min 400) (max 100))]
+                attr-col-width (->> attributes count (/ 1000) int (min 400) (max 100))
+                process-local-state (and (:process-local-state props) (persistence/check-saved-state {:key (str (common/workspace-id->string (:workspace-id props)) ":data:" (:selected-entity-type @state))}))]
```

Comment:
What is this value supposed to be? It seems like the idiomatic option here would be a `when`. Also, maybe a line break in here.